about.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/aboutstyles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> 
+    <title>About Women Tech</title>
+</head>
+<body>
+    <header class="navigation">
+        <nav class="nav-list">
+          <div class="header">
+            <ul>
+                <li><i class='fa fa-facebook-official'></i></li>
+                <li><i class='fa fa-twitter'></i></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">Mypage</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+            <div class="logo">
+              <p class="logo-pic">women tech <br> network</p>
+            </div>
+         <div class="menu">
+            <a href="#"><img src="img/menu.png" alt="menu icon"></a>
+         </div>
+         <div class="mobile-menu display-mobile">
+        <div class="close-menu">
+         <a href="#"><img src="img/Close.png" alt="close-menu"></a>
+        </div>
+         <ul>
+           <li><a class="menu-links" href="index.html">Home</a></li>
+           <li><a class="menu-links" href="#">Program</a></li>
+           <li><a class="menu-links" href="#">Join</a></li>
+           <li><a class="menu-links" href="#">Sponser</a></li>
+           <li><a class="menu-links" href="#">News</a></li>
+           <li><a class="menu-links-last" href="#">CC Campaign</a></li>
+         </ul>
+        </div>
+        </nav>
+
+        <div class="desktop-menu">
+          <div class="logo">
+             <a href="index.html"><img src="/img/logo.PNG" alt="logo"></a> 
+            </div>
+            <div class="menu-list">
+              <ul>
+                  <li><a class="menu-links" href="index.html">Home</a></li>
+                  <li><a class="menu-links" href="#">Program</a></li>
+                  <li><a class="menu-links" href="#">Join</a></li>
+                  <li><a class="menu-links" href="#">Sponsor</a></li>
+                  <li><a class="menu-links" href="#">News</a></li>
+                  <li><a class="menu-links-last" href="#">CC Campaign</a></li>
+                </ul>
+               </div>
+            </div>
+    </header>
+
+    <section class="introduction">
+        <h2>"Hello! Sharing World"</h2>
+        <h1>WOMEN IN TECH GLOBAL CONFERENCE 2023</h1>

```

Comment:
In a section, h1 should never be preceded with another heading with less importance. Consider making your `h2` heading a `p` tag.  Do the same for the index.html

Reply:
Okay, thank you for reviewing my project. I'll correct it.